tools/db_bench_tool.cc

```diff
@@ -7119,6 +7131,41 @@ int db_bench_tool(int argc, char** argv) {
     FLAGS_env = new ROCKSDB_NAMESPACE::HdfsEnv(FLAGS_hdfs);
   }
 
+#ifdef IPPCP
```

Comment:
Can this be done via EncryptionProvider::CreateFromString rather than here?  It would make it easier to use the provider in different scenarios